### PR TITLE
Issue#6 fixed

### PR DESCRIPTION
**Pull request by NeerajChatterjee**
Added the functionality to upload images from the PC.

### CODE REVIEW

File: src/components/Meme.js [hunk 1]
```diff
@@ -57,6 +77,10 @@ export default function Meme() {
 				<button className="form__button" onClick={getRandomMeme}>
 					Generate Meme
 				</button>
+				<label for="image-upload" className="form__button upload_image__button">
+					Upload Meme Image
+				</label>
+				<input id="image-upload" type="file" onChange={uploadImage} />
```
Comment (dev-lovedeep) [inline src/components/Meme.js:83]: add  accept attribute in the input tag here

File: src/components/Meme.js [hunk 2]
```diff
@@ -39,6 +41,24 @@ export default function Meme() {
 			[name]: value,
 		}));
 	}
+
+	// this is for uploading the image from the PC
+	function uploadImage(event){
+		console.log(event.target.files[0].type);
+
+		// accepts image in the form of PNG/JPG/JPEG
+		if (event.target.files[0].type == "image/png" || event.target.files[0].type == "image/jpg" || event.target.files[0].type == "image/jpeg"){
```
Comment (dev-lovedeep) [inline src/components/Meme.js:50]: use '===' instead of '=='

File: src/components/Meme.js [hunk 3]
```diff
@@ -23,6 +23,8 @@ export default function Meme() {
 		url: "",
 	});
 
+	const [file, setFile] = useState()
```
Comment (dev-lovedeep) [inline src/components/Meme.js:26]: this is unused

**Review by dev-lovedeep — COMMENTED** [review]
make these small changes and do not use separate commit for updating contributors.md file

**Comment by NeerajChatterjee** [comment]
Fixed the mentioned issues and extra commit for contributors.md file is regretted.

**Comment by dev-lovedeep** [comment]
fixes issue #6

**Pull request merged**